### PR TITLE
urlwatch: update to 2.23 and claim maintainership

### PR DESCRIPTION
**Pull request by RunningDroid**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 
(Please choose at least one native build and, if supported, at least one cross build. More are better.)
- [x] I built this PR locally for my native architecture, (x86-64 GLIBC)
- [x] I built this PR locally for these architectures (all crossbuilds):
  - [x] aarch64-musl
  - [x] armv7l
  - [x] armv6l-musl

### CODE REVIEW

**Comment by RunningDroid** [comment]
I forgot to run the tests, I'm fixing them now.

**Comment by RunningDroid** [comment]
My computer died, so it might be a day or 2 before I can get back to this.

File: srcpkgs/urlwatch/template [hunk 1]
```diff
@@ -11,11 +11,11 @@ depends="python3-appdirs python3-keyring python3-minidb python3-requests
 checkdepends="python3-pytest python3-pycodestyle python3-docutils
  python3-Pygments ${depends}"
```
Comment (ericonr) [inline srcpkgs/urlwatch/template:12]: Add `jq` here (or even in `depeds`) and that part should work.

https://urlwatch.readthedocs.io/en/latest/dependencies.html lists some other optional dependencies.

Comment (FollieHiyuki) [reply srcpkgs/urlwatch/template:12]: That doesn't work though. What it needs is the jq Python bindings [jq.py](https://github.com/mwilliamson/jq.py), not `jq` itself. We can add it as a new package.

**Comment by RunningDroid** [comment]
I've got python3-jq packaged now but ld doesn't want to link it against libjq for some reason, If I add "LD_PRELOAD=/usr/lib/libjq.so.1.0.4" and add jq-devel to urlwatch's checkdepends all the tests pass but I don't think that should be merged.

**Comment by ericonr** [comment]
New packge commit message should include version: `New package: pkgname-version`

**Comment by ericonr** [comment]
I think we should skip tests for urlwatch and not package `python-jq`, if you want to avoid dealing with the mess... I can try to look into it at some point, but I'm not sure it's worth it, because we'll always be fighting upstream to package it without bundling everything in...

**Comment by RunningDroid** [comment]
I'm gonna have to agree, I've basically hit a dead end with python3-jq and there aren't any other distro's packages that I can copy from: https://repology.org/project/python:jq/versions

**Pull request merged**